integration_tests/pages/barcode/createNewContact.ts

```diff
@@ -13,21 +13,21 @@ export default class CreateNewContactPage extends Page {
     this.prisonerNameField().should('be.focused')
   }
 
-  submitWithValidValues = (): ReviewRecipientsPage => {
-    this.enterAValidPrisonerName()
-    this.typeAheadAValidPrison()
+  submitWithValidValues = (prisonerName = 'Gage Hewitt', prisonName = 'ashfield'): ReviewRecipientsPage => {
```

Comment:
Minor refactoring of these methods to give them args with default values. It means the existing tests can call these methods as they do currently (ie. where we dont really care _who_ is added), but for the Legal Sender E2E we want to add several recipients and for them all to be different.